client/build-config/src/webpack/babel-loader.ts

```diff
@@ -0,0 +1,11 @@
+import path from 'path'
+
+import { ROOT_PATH } from '../paths'
+
+export const babelLoader = {
```

Comment:
Let's keep loaders wrapped into functions to be able to customize them later and be consistent with other loaders:
```suggestion
export const getBabelLoader = (): RuleSetRule => ({
```

@gitstart-sourcegraph, let's use the same pattern for plugins too.